source/_patterns/03-organisms/gallery/gallery-overlay.twig

```diff
@@ -28,11 +28,20 @@
             }
           %}
           <div class="u-flex-wrap-break"></div>
-          {% if loop.index == 3 %}
+
+          {% if loop.index is divisible by(3) and loop.index <= 6 %}
             <div class="l-wrap">
               {% include 'molecules-billboard' %}
             </div>
           {% endif %}
+
+          {# Ad requirements for greater than 10 total items #}
+          {% if loop.length > 10 and loop.index > 10 and loop.index is divisible by(6) %}
+            <div class="l-wrap">
+              {% include 'molecules-billboard' %}
+            </div>
+          {% endif %}
```

Comment:
I discussed the ad cadence with Kate and I think we've managed to simplify the description of the ad conditions. Basically it's "show ads after every 6th slide, and also show one after the 3rd slide". No more different conditions based on the total number of ads. I think we can probably simplify the code with this logic too.

I'm also going to update this on the tickets.

e.g.
```
1
2
3 (three)
***AD***
4
5
6 (multiple of 6)
***AD***
7
8
9
10
11
12 (multiple of 6)
***AD***
13
14
15
16
17
18 (multiple of 6)
***AD***
19
20
21
22
23
24 (multiple of 6)
***AD***
25
...
```

Reply:
@walsh9 This is what it's doing right now - https://nypr.southleft.com/?p=pages-article-gallery-long

That extra piece of logic is to break out of the "every 3" and move to an "every 6".

Reply:
i was just thinking you could simplify to a single block. something like:
```
{% if loop.index is divisible by(6) or loop.index = 3 %}
```

Reply:
Ah, I see. That's much cleaner. Updated!